### DISCUSSION TITLE
Not able to access sling server

### DISCUSSION THREAD

**Discussion by Nli10me** [GerryDazoo/Slinger#245] [General]
Hello,

Thank you Gerry for your brilliant code to extend the usage of our Slingboxes.

I am, and have been in a dilemma for several days now (combing through the discussions) and really didn't want to trouble you or the community with my problems, but the day is drawing near and I am getting desperate. I am not knowledgeable (at all) on this kind of stuff and am having a very hard time even understanding some of the terminology used. So please be understanding.

I have done all what is called for with setting up the sling server(config file). I am trying to remotely set up and access my SB. I have entered (uncommented) my admin password, my public ip, port number and [SERVER] port=8080.

 I execute the slingbox_server.exe file:

Version : 3.08c Running on Windows-10-10.0.19044-SP0 pid= 7748
Using config file config.ini
Connection Manager Running on port 8080 with 2 max streams using URL slingbox.
Streamer Running:  2 config.ini SLINGBOX 8080 8080 8388608
starting up on  port 8080 8388608
Checking for slingbox at ('xx.xx.xx.xxx', 5201)
('xx.xx.xx.xxx', 5201) OK
8080 Using slingbox at  ('xx.xx.xx.xxx', 5201)
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Reading Custom Remote definition from remote.txt
 * Serving Flask app 'slingbox_server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off

 I can ONLY get a stream with the computer that I executed the sling_server.exe with via VLC network stream using localhost or 127.0.0.1 (http://127.0.0.1:8080/slingbox). I get a very clear video with clean audio of my slingbox broadcast, but can't change the channel. (I have left the SB on so I could access it with VLC)

I've tried to access it from another computer with VLC and get an error saying "VLC is unable to open the MRL...". OBS also doesn't connect. I've tried to entering "http://xx.xx.xx.xxx:8080/Remote" in browser, too. I've tried all of them with finderid enabled and disabled multiple times with multiple adjustments, but got the same outcome. 

I have already created exclusions for Defender (before initiating sling_server.exe) and also have configured the firewall (in troubleshooting) and have even tried to turn it completely off and tried to access the sling server from my other computer with no success via VLC (MLR error) and in browser "http://xx.xx.xx.xxx:8080/Remote" (not found).

I have also tried port=8081 and 8082 just to see if it would have any affect (but didn't think so) and they had the same outcome (only localhost and 127.0.0.1 got access).

I have seen another person having problems  in another discussion talking about IPx6? I have no idea if that is affected me, but... could it be?

Any suggestions or help is welcomed. Sorry for such a long explanation and thank you.

**Comment by GerryDazoo** [comment]
Turn off your firewall temporarily to see if that fixes it. If so, turn it back on and then add an input rule to open TCP port 8080.

**Reply by AndyJStubbs** [reply]
Just to determine if the issue is actually with vlc or not what happens if you type the full url into the browser - it should start to download the stream. If that fails to connect then it indicates the problem is more likely firewall or network related.

**Reply by GerryDazoo** [reply]
Mikey, it can't be your double nat because he says he can remote access the SB when running the server locally.

**Reply by Nli10me** [reply]
Thank you so much for your quick replies. I am sorry I could not reply quicker.

I must apologize for typing a mistake. It is not IPx6, but IPv6.

I also didn't try access the remote with 127.0.0.1 previously because localhost failed in the browser (same as server), and had automatically thought it would be the same outcome. With THAT being said, I can access the remote in browser with 127.0.0.1 and turn on/off and switch channels. So, this calmed me down a lot. Even if I can't get access from another computer when the server is up and running, at the bare minimum, I can still watch from the server itself.

I will mess around with it more in the days to come to see if I can get it running where I can access the server with another computer in or outside my home network. 

So as I stand now, my computer that I set up as the server (with Gerry's code) is the only computer that I can access and watch my SB. No other device and access it even when the server is up and running. I don't know why, but I know that the code works and I can still watch from my computer. If I NEED to, I can tether my laptop via smartphone and I can access my SB, but I don't call lugging a laptop around to watch my SB very mobile (anymore). lol

I am happy that my SB still has life. I am extremely grateful, Gerry! Thank you!

**Reply by AndyJStubbs** [reply]
That has to be a firewall issue surely not allowing port 8080 outbound access

**Reply by Nli10me** [reply]
"That has to be a firewall issue surely not allowing port 8080 outbound access" Thank you, Andy.

That's what I was thinking this whole time, but couldn't find a solution. I have noticed that the slingplayer is not working from today which can only mean that the servers have been cut. However, I was able to access my SB from my slinger/computer. I can't thank you enough, Gerry.

I will try to seek out the connection problem later this weekend if time permits. I am kind of interested to see if I can eventually stream my SB via my smartphone. I have seen an "android" discussion, but have not looked at it yet.
